### ISSUE TITLE
[Feature Request] - Add option to hide specific buttons without hiding all

### ISSUE THREAD

**Issue by Emma-DreamAgility** [kamranahmedse/driver.js#114]
Hey, thanks for this project - it's proving really useful for me at the moment.

One small request is that you add in an option to hide specific buttons from individual steps. I'm often in a position where I want to hide the "Close" and/or "Prev" buttons while retaining the Next button. Right now I'm having to jump into the driver code and remove these buttons from the html but it'd be nice to have an officially supported method for hiding them.

Thanks again :)

**Comment by kamranahmedse** [comment]
There is already an option for that. Just set the `showButtons` to false in any given step and it will not be hidden for the rest of the steps

```javascript
[
    {
      element: document.getElementById('driver-demo-head'),
      popover: {
        title: 'Before we start',
        description: 'This is just one use-case, make sure to check out the rest of the docs below.',
        nextBtnText: 'Okay, Start!',
      },
    }, {
      element: '#logo_img',
      popover: {
        title: 'Focus Anything',
        description: 'You can use it to highlight literally anything, images, text, div, span, li etc.',
        showButtons: false  // hide buttons for this step
      },
    }, {
      element: '#name_driver',
      popover: {
        title: 'Why Driver?',
        description: 'Because it lets you drive the user across the page',
      }
    }
]
```

**Comment by Emma-DreamAgility** [comment]
That option hides all of the buttons though right? Like if showButtons is false the Next, Prev and Close buttons are all hidden. I was suggesting something more like:

`{
      element: '#logo_img',
      popover: {
        title: 'Focus Anything',
        description: 'You can use it to highlight literally anything, images, text, div, span, li etc.',
        showNextButton: true,
        showPrevButton: true,
        showCloseButton: false
      },
 }`

If you didn't want a user to be able to close the tutorial before the last step for example, the only option currently (that I'm aware of) would be to alter the driver source code.

**Comment by AnkitaIn** [comment]
Is there any update on this request? Is this included in your road map?
Also instead of "close" button, "close" icon can be used?

**Comment by AnkitaIn** [comment]
@kamranahmedse Is there any update on this request? 
Is this feature request present in your road map?

**Comment by theoo360** [comment]
@AnkitaIn @Emma-DreamAgility 
Modified Driver.js with option to hide any button (configurable for any element individually)

example:
```
{  
    element: '#logo_img',
      popover: {
        title: 'Title on Popover',
        description: 'Body of the popover',
        showNextButton: true,
        showPrevButton: false,
        showCloseButton: false
      },
 }
```

txt file because cannot add JS here:
[driver.min.js.txt](https://github.com/kamranahmedse/driver.js/files/5215001/driver.min.js.txt)
